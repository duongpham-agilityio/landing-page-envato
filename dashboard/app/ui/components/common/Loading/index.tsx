'use client';

import { Flex, Spinner } from '@chakra-ui/react';

const Loading = () => (
  <Flex justifyContent="center" p={5}>
    <Spinner />
  </Flex>
);

export default Loading;
