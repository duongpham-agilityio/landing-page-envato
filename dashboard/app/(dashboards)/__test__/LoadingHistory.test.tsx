// Sections
import Loading from '../histories/loading';

const { render } = testLibReactUtils;

describe('Loading render', () => {
  test('Should render match with snapshot.', () => {
    const { container } = render(<Loading />);

    expect(container).toMatchSnapshot();
  });
});
