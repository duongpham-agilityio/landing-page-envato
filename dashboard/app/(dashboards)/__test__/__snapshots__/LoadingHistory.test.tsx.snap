// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading render Should render match with snapshot. 1`] = `
<div>
  <div
    class="chakra-spinner css-1f8r8ah"
  >
    <span
      class="css-8b45rq"
    >
      Loading...
    </span>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
